docker/gcp_pudl_etl.sh

```diff
@@ -109,6 +120,9 @@ if [[ $ETL_SUCCESS == 0 ]]; then
     ETL_SUCCESS=${PIPESTATUS[0]}
 
     # Dump outputs to s3 bucket if branch is dev or build was triggered by a tag
+    # TODO: this behavior should be controlled by on/off switch here and this logic
```

Comment:
Agree - having the action just pass in a switch would be nice. I think at some point we should replace this whole nightly build harness with a Python script that's more robust and that would be a nice time to fix this too.